fluent/reference.md

```diff
@@ -618,14 +618,57 @@ $builder->joinedTableInheritance()->column('type');
 ```
 
 <a name="lifecycle-callbacks"></a>
-## Lifecycle callbacks
+## Lifecycle events and callbacks
 
-Doctrine allows configuring specific methods of an entity to be called as listeners of
-_lifecycle events_. This lets you easily hook into specific moments of the Doctrine entity lifecycle.
+Doctrine allows hooking into its lifecycle via [Events](https://doctrine-orm.readthedocs.io/projects/doctrine-orm/en/latest/reference/events.html). Fluent provides a bridge to hook into this event system as well.
 
-Use `$builder->events()` to access the `LaravelDoctrine\Fluent\Builders\LifecycleEvents` object.
+There are two sets of events that can be used by two different types of listeners:
+
+**Lifecycle Event** Listeners and Subscribers are classes with specific callback methods that receives some kind of EventArgs instance.
+
+**Lifecycle Callbacks** are methods on the entity classes that are called when the event is triggered.
```

Comment:
@FoxxMD if you change phrasing as suggested above, maybe this one could change as well:

> Entities themselves can listen to **Lifecycle Callbacks** and have a method called on them when a specific event is triggered.